### PR TITLE
Admin Panel Users

### PR DESCRIPTION
**Pull request by kfungwm**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,12 +8,24 @@ const express = require('express'),
       bcrypt = require('bcrypt'),
       morgan = require('morgan');
 
+var app = express(),
+    db = require('./models');
+
+
+
+// var registrationRouter = require('./routes/registration');
```
Comment (izelnakri) [inline index.js]: remove unused code

File: index.js [hunk 2]
```diff
@@ -35,13 +47,24 @@ app.use(session({
...
+
 app.use('/', userRouter);
 
 app.use('/', authenticationRouter);
 
 app.use('/', movieRouter);
 
 
+
+// app.get('/', (request,response) => {
```
Comment (izelnakri) [inline index.js]: remove unused code

File: index.js [hunk 3]
```diff
@@ -50,6 +73,13 @@ app.get('/users/:id', (req, res) => {
    // this will be other users profile
 });
 
+
+
+//
+// app.get('/logout', (req, res) => {
```
Comment (izelnakri) [inline index.js]: remove unused code

File: routes/admin.js [hunk 4]
```diff
@@ -15,32 +15,45 @@ var express = require('express'),
 //   });
 // });
 
-router.get('/admin/movies/new', (request,response) => {
+router.get('/movies/new', (request,response) => {
   response.render('admin/movies/new');
 });
 
-router.get('/admin/users/adminpanel', (request,response) => {
+router.get('/users/adminpanel', (request,response) => {
```
Comment (izelnakri) [inline routes/admin.js:22]: what is /users/adminpanel link?

File: routes/user.js [hunk 5]
```diff
@@ -27,16 +27,29 @@ const bcrypt = require('bcrypt'),
...
 router.get('/profile', (req, res) => {
    db.User.findOne({
-      where: {
-         id: req.session.user.id
-      }
+     where: {
+       id:req.session.user.id
+     }
+   }).then((user) => {
+      res.render('users/profile', { user: req.session.user, userprofile: user });
```
Comment (izelnakri) [inline routes/user.js]: what is userprofile? seems to me it is the same as the req.session.user? or just use the user variable from the database without the user, they refer to the same entity

File: routes/user.js [hunk 6]
```diff
@@ -27,16 +27,29 @@ const bcrypt = require('bcrypt'),
...
+});
+
+router.get('/profile/edit', (req, res) => {
+   db.User.findOne({
+     where: {
+       id: req.params.id
+     }
    }).then((user) => {
-      res.render('users/profile', { user: req.session.user, userProfile: user });
+      res.render('users/edit', { user: req.session.user, userprofile: user });
```
Comment (izelnakri) [inline routes/user.js]: what is userprofile? seems to me it is the same as the req.session.user? or just use the user variable from the database without the user, they refer to the same entity

File: routes/user.js [hunk 7]
```diff
@@ -60,7 +73,7 @@ router.put('/profile', (req, res) => {
 
 
 router.get('/edit-password', (req, res) => {
-   res.render('users/edit-password', { user: req.session.user });
+   res.render('users/edit-password', { user: req.session.user, userprofile: user });
```
Comment (izelnakri) [inline routes/user.js]: what is userprofile? seems to me it is the same as the req.session.user? or just use the user variable from the database without the user, they refer to the same entity

File: views/admin/users/adminpanel.pug [hunk 8]
```diff
@@ -2,18 +2,20 @@ extends ../../application.pug
 
 block content
   h1 Users List
+  each user in users
   .card.card-block
     .row
       .col-md-4.hidden-md-down
-        img(src=users.imageURL alt=users.title + "Cover Image" class="img-fluid")
+        img(src=user.profileImageURL width="350" height="350" float-xs-left)
```
Comment (izelnakri) [inline views/admin/users/adminpanel.pug]: class="float-xs-left"

File: routes/user.js (hunk 6 above)
Comment (kfungwm) [reply routes/user.js]: I dont' know Marie made it

File: routes/user.js (hunk 5 above)
Comment (kfungwm) [reply routes/user.js]: I dont' know Marie made it

File: routes/admin.js (hunk 4 above)
Comment (kfungwm) [reply routes/admin.js:22]: Wait for the login of admin then I can make the files.

**Pull request merged**